src/migrations/20250405143306-remove-similarity-tables.js

```diff
@@ -0,0 +1,35 @@
+const { DataTypes } = require('sequelize');
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      await queryInterface.dropTable('ZALGoalSimilarityGroupGoals', { transaction });
+      await queryInterface.sequelize.query('DROP FUNCTION IF EXISTS "public"."ZALNoUpdateFGoalSimilarityGroupGoals" ()', { transaction });
```

Comment:
I guess if we are removing all old goals makes sense to drop all the audit info (if).